standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/dataconnector/jdbc/AbstractJDBCConnectorProvider.java

```diff
@@ -415,4 +438,7 @@ protected Properties getConnectionProperties() {
     connectorPropMap.forEach((k, v) -> connectionProperties.put(k, v));
     return connectionProperties;
   }
-}
\ No newline at end of file
+
+  @Override
+  protected String getDatasourceType() { return type; };
```

Comment:
I think there is a semi-colon after } which isn't required

Reply:
fixed